src/main/java/com/zaxxer/hikari/pool/ProxyConnection.java

```diff
@@ -55,9 +55,9 @@
 
    protected Connection delegate;
 
-   private final PoolEntry poolEntry;
-   private final ProxyLeakTask leakTask;
-   private final FastList<Statement> openStatements;
```

Comment:
Remove this change, it does not accelerate GC at all because these are backward references toward the GC root.